### DISCUSSION TITLE
How do i use this tool with Neovim

### DISCUSSION THREAD

**Discussion by Nizarll** [JohnnyMorganz/luau-lsp#490] [Q&A]
Everything is in the title

**Comment by JohnnyMorganz** [comment]
See https://github.com/lopi-py/luau-lsp.nvim

**Comment by Nizarll** [comment]


**Reply by Nizarll** [reply]
here's my mason-lspconfig

**Reply by Nizarll** [reply]
![Screenshot 2023-11-19 200338](https://github.com/JohnnyMorganz/luau-lsp/assets/151186173/a093f18a-995c-443e-b86d-2aacbe7cd702)
_
